### ISSUE TITLE
Unable to reproduce the result reported in paper (Transformer and MOTIFS)

### ISSUE THREAD

**Issue by Nora-Zhang98** [dongxingning/SHA-GCL-for-SGG#6]
Hi author, thanks for your excellent work. I am trying to reproduce your work but I find the mR@K metric cannot achieve the results in your paper. I test SHA-GCL and MOTIFS-GCL for predcl, but they are all lower than the metrics in your paper. 
For SHA-GCL, I train 60000 iters, base lr 0.001, batch size 16. The result is for SHA-GCL. Extra hyperparameters are shown as below. Could you please share your hyperparameters so I can reimplement your work? Thanks!

<img width="646" alt="无标题2" src="https://user-images.githubusercontent.com/60804365/178128646-4045d368-d8a1-4f58-b737-3b10a29f1cd6.png">

<img width="654" alt="无标题" src="https://user-images.githubusercontent.com/60804365/178128644-a3c07985-5e9c-4026-baa7-a3487f0b7457.png">

**Comment by dongxingning** [comment]
Well, that is quite strange. It seems that your reproduced results are much lower than ours (both Recall and Mean Recall). Do you change some parts of our code? I guess the main reason may not lie in the hyper-parameters.

**Comment by dongxingning** [comment]
Maybe you can run the configuration "MOTIFS" without any GCL decoder (MOTIFS baseline in PredCls)? If your result on MOTIFS-PredCls is normal, you can zip your code and send me to check, thanks a lot.

**Comment by Nora-Zhang98** [comment]
I will follow your suggestion to test again. Thank you for your suggestion!

**Comment by Nora-Zhang98** [comment]
Dear author, I follow your suggestion and run MOTIFS without GCL for PredCl. The results are normal. How should I send you the code? Thanks!
<img width="605" alt="无标题" src="https://user-images.githubusercontent.com/60804365/178900922-91e13e5b-40d5-4c97-ab53-b16604401a56.png">

**Comment by dongxingning** [comment]
You can email me at 1463365882@qq.com (recommended) or dongxingning1998@gmail.com, thanks!

**Comment by Nora-Zhang98** [comment]
I have already sent my code to your gmail. Thanks for your help.

**Comment by Nora-Zhang98** [comment]
Oh I didn't notice the **recommanded**, I sent code again to your qq emial.

**Comment by dongxingning** [comment]
Well, I have checked your log files of your reproduced SHA-GCL; it seems that you set the batch size to 2 rather than your claimed 16. Thus, though you run 60000 steps, it is actually equal to 15000 iters at the setting of batch 8, the model does not converge at these steps, so it is very normal that your result is much lower than us. Besides, we downloaded the code from Github and re-runed the model again. Here is our result on the validation set at 45000 steps, and we stopped it because it proves that our results could be reproduced.
![1657973349371](https://user-images.githubusercontent.com/58069157/179354306-397e581f-314a-450b-b475-d22a5755b276.png)
